server/implementation/src/main/java/io/smallrye/graphql/execution/error/ExecutionErrorsService.java

```diff
@@ -37,6 +35,7 @@ public class ExecutionErrorsService {
     private static final Jsonb JSONB = JsonbBuilder.create(new JsonbConfig()
             .withNullValues(Boolean.TRUE)
             .withFormatting(Boolean.TRUE));
+    private static final ErrorExtensionProviders ERROR_EXTENSION_PROVIDERS = new ErrorExtensionProviders();
```

Comment:
Can we avoid this being static? That would cause issues in scenarios where multiple GraphQL apps run in the same JVM (in WildFly, for example) because all apps would share the same value here, and therefore I envision that

- we would lose the ability for each application to have different providers
- if a provider is provided in the JAR of one application, attempt of another application to use it would lead to class loading problems

I think this field should just work if you remove `static`, no?

Reply:
+1

Reply:
agree， i didn't pay attention that ExecutionErrorsService is static in ExecutionResponse